dialog/src/main/java/roboy/dialog/states/devoStates/QuestionRoboyQAState.java

```diff
@@ -268,61 +227,43 @@ private String inferPasAnswer(Map<SemanticRole, String> pas, Roboy roboy) {
         if (matchPas(pas, new Pair(SemanticRole.AGENT, "old")) || matchPas(pas, new Pair(SemanticRole.PATIENT, ".*\\bage\\b.*"))) {
             answer = extractAge(roboy);
         } else if (matchPas(pas, new Pair(SemanticRole.PREDICATE, "from"))) {
-            answer = extractNodeNameForPredicate(Neo4jRelationship.FROM, roboy);
-        }
-        else if (matchPas(pas, new Pair(SemanticRole.LOCATION, "where"), new Pair(SemanticRole.PREDICATE, "live"))) {
+            extractNodeNameForPredicate(Neo4jRelationship.FROM, roboy);
+        } else if (matchPas(pas, new Pair(SemanticRole.LOCATION, ".*"))) {
             answer = extractNodeNameForPredicate(Neo4jRelationship.LIVE_IN, roboy);
-        }
-//        else if (matchPas(pas, new Pair(SemanticRole.AGENT, "you"), new Pair(SemanticRole.MANNER, "how"))) {
-//            answer = "Yo moma says I am a good boy!";
-//        }
-        else if (matchPas(pas, new Pair(SemanticRole.AGENT, "who"))) {
-            if (matchPas(pas, new Pair(SemanticRole.PATIENT, "you"), new Pair(SemanticRole.PREDICATE, "are"))) {
-                answer = extractNodeNameForPredicate(Neo4jProperty.name, roboy) + " "
-                        + extractNodeNameForPredicate(Neo4jProperty.identitiy, roboy);
+        } else if (matchPas(pas, new Pair(SemanticRole.AGENT, "you"), new Pair(SemanticRole.MANNER, "how"))) {
+            answer = "Yo moma says I am a good boy!";
+        } else if (matchPas(pas, new Pair(SemanticRole.AGENT, "who"))) {
+            if (matchPas(pas, new Pair(SemanticRole.PATIENT, "you"))) {
+                answer = extractNodeNameForPredicate(Neo4jProperty.full_name, roboy);
             } else if (matchPas(pas, new Pair(SemanticRole.PATIENT, ".*\\b(father|dad)\\b.*"))) {
                 answer = extractNodeNameForPredicate(Neo4jRelationship.CHILD_OF, roboy);
             } else if (matchPas(pas, new Pair(SemanticRole.PATIENT, ".*\\b(sibling|brother)\\b.*"))){
                 answer = extractNodeNameForPredicate(Neo4jRelationship.SIBLING_OF, roboy);
-            } else if (matchPas(pas, new Pair(SemanticRole.PREDICATE, "created|made|built"), new Pair(SemanticRole.PATIENT, "you"))) {
-                answer = extractNodeNameForPredicate(Neo4jRelationship.CREATED_BY, roboy);
             }
         } else if (matchPas(pas, new Pair(SemanticRole.PREDICATE, "do|like"), new Pair(SemanticRole.AGENT, "you"))){
-//            double prob = Math.random();
-//            if (prob < .3) {
-//                answer = extractNodeNameForPredicate(Neo4jProperty.abilities, roboy);
-//            } else if(prob < .7) {
-//                answer = extractNodeNameForPredicate(Neo4jRelationship.HAS_HOBBY, roboy);
-//            } else {
+            double prob = Math.random();
```

Comment:
no, this doesnt go in

Reply:
@Waguramu @josephbirkner do you recall why we added this to this class